examples/example.rs

```diff
@@ -12,7 +12,7 @@ fn main() {
         println!("No previous history.");
     }
     loop {
-        let readline = rl.readline(">> ");
+        let readline = rl.readline("\x1b[1;32m>>\x1b[0m ");
```

Comment:
No, it is not necessary.
It just shows that rustyline properly handles prompt with ansi escape codes.